packages/electrode-react-webapp/lib/react-webapp.js

```diff
@@ -104,14 +104,23 @@ function makeRouteHandler(routeOptions, userContent) {
       }
 
       const chunkNames = chunkSelector(request);
-      const devCSSBundle = chunkNames.css
-        ? `${devBundleBase}${chunkNames.css}.style.css`
-        : `${devBundleBase}style.css`;
+
+      let devCSSBundle;
+      if (chunkNames.css) {
+        const cssChunks = Array.isArray(chunkNames.css)
+          ? chunkNames.css
+          : [ chunkNames.css ];
+        devCSSBundle = _.map(
+          cssChunks,
+          (chunkName) => `${devBundleBase}${chunkName}.style.css`
+        );
+      } else devCSSBundle = [`${devBundleBase}style.css`];
+
       const devJSBundle = chunkNames.js
         ? `${devBundleBase}${chunkNames.js}.bundle.dev.js`
         : `${devBundleBase}bundle.dev.js`;
       const jsChunk = _.find(assets.js, asset => _.includes(asset.chunkNames, chunkNames.js));
-      const cssChunk = _.find(assets.css, asset => _.includes(asset.chunkNames, chunkNames.css));
```

Comment:
I am not sure what's the reason behind this line change here, and it seems it's a breaking change.

Reply:
Hmm, my earlier comment explaining why I did it seems to have been lost. I was asking what its purpose was because it seemed to actually have an adverse affect.

Reply:
I restored the filtering and confirmed it works both when `chunkNames.css` is an array or just a string.